draft-tschofenig-rats-psa-token.md

```diff
@@ -513,14 +513,19 @@ in favor of other information.
 {: #sec-profile-definition-claim}
 
 The Profile Definition claim encodes the unique identifier that corresponds to
-the EAT profile described by this document.  This allows a receiver to assign
+one of the EAT profiles described by this document.  This allows a receiver to assign
 the intended semantics to the rest of the claims found in the token.
 
-The EAT `profile` (claim key 265) is used.  The following constraints
-apply to its type:
+The EAT `profile` (claim key 265) is used.
 
-* The URI encoding MUST be used.
-* The value MUST be `http://arm.com/psa/2.0.0`.
+The URI encoding MUST be used.
+
+The value MUST be one of:
+
+* `tag:psacertified.org,2023:psa#a+es` for the profile defined in {{sec-ecdsa-profile}},
+* `tag:psacertified.org,2023:psa#s+hmac` for the profile defined in {{sec-hmac-profile}}.
+
+Future profiles derived from the "base" PSA profile SHALL mint their unique value as described in {{sec-profile-uri-structure}}.
```

Comment:
Should we be consistent and use baseline (instead of "base")?

Reply:
👍 fixed